shells/tests/arcs/ts/runtime/arc-test.ts

```diff
@@ -25,7 +25,7 @@ import '../../../../lib/arcs-ui/dist/install-ui-classes.js';
 
 describe('Arc', () => {
   afterEach(() => {
-    DriverFactory.clearRegistrationsForTesting();
+    Runtime.resetDrivers();
```

Comment:
the original method has `ForTesting` suffix. is the updated method intended not only for testing?